README.md

```diff
@@ -139,14 +139,15 @@ and the type of the change.
 ```
 >> Please specify the category of your change
 
-1. New feature
-2. Bug fix
-3. Feature change
-4. New deprecation
-5. Feature removal
-6. Security fix
-7. Performance improvement
-8. Other
+1. New feature (minor)
+2. Bug fix (patch)
+3. Breaking change (major)
+4. New deprecation (minor)
+5. Feature removal (major)
+6. Security fix (minor)
+7. Performance improvement (minor)
+8. Dependencies updated (patch)
+9. Other (minor)
```

Comment:
Instead of showing here the version number each category is going to modify, I would show the code for that category because it's what users need in case they to edit the file manually.
What do you think?

Reply:
OK, that's fine. Do I include what version number each category will modify in the README? I think that part is not documented.

Reply:
Yes, we can add that too.